plugins/topology/README.md

```diff
@@ -207,6 +207,27 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
     backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
   ```
 
+  You can add the `backstage.io/kubernetes-label-selector` annotation if you have multiple entities and Red Hat OpenShift Dev Spaces is configured.
+
+  ```yaml title="catalog-info.yaml"
+  annotations:
+    backstage.io/kubernetes-label-selector: 'component in (<BACKSTAGE_ENTITY_NAME>,che)'
+  ```
+
+  If you are using the above label selector ensure that you add the following labels to your resources
+
+  ```yaml title="resource.yaml"
```

Comment:
`resource.yaml` seems confusing to me, we can just say `The following label is added to the resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity`